db/pre-re/mob_db.yml

```diff
@@ -40645,42 +40645,49 @@ Body:
     SkillRange: 10
     ChaseRange: 12
     Size: Large
-    Race: Formless
+    Race: Brute
     Element: Fire
     ElementLevel: 3
     WalkSpeed: 200
-    AttackDelay: 1150
-    AttackMotion: 1150
-    DamageMotion: 288
+    AttackDelay: 1152
+    AttackMotion: 1152
+    DamageMotion: 576
     Ai: 21
     Class: Boss
     Modes:
       Mvp: true
+    MvpDrops:
+      - Item: Old_Violet_Box
+        Rate: 5500
+      - Item: Old_Violet_Box
+        Rate: 5000
+      - Item: Old_Card_Album
+        Rate: 2000
```

Comment:
This is the only difference I've found compared with the data I got from bRO back in the day, mine says the rate for OCA is 3000, but can't remember where that came from, probably from bRO forums.